### DISCUSSION TITLE
Welcome to react-popper-tooltip Discussions!

### DISCUSSION THREAD

**Discussion by mohsinulhaq** [mohsinulhaq/react-popper-tooltip#105] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KCN07** [comment]
Is there any props to make elements inside tooltip clickable? Or is it possible to make inside elements clickable? How to solve this issue? Can't using elements inside tooltip in version 4.3.1

**Reply by mohsinulhaq** [reply]
@KCN07 The elements inside should already be clickable. Can you please open an issue with a reproducer codesandbox that showcases your problem?

**Reply by KCN07** [reply]
Here the basic code I tried from npm documentation. But when I tried to insert a button inside tooltip the button isn't clickable.
https://codesandbox.io/s/peaceful-cannon-4jceli?file=/src/App.js

**Reply by mohsinulhaq** [reply]
@KCN07 pass `interactive: true` to userPopperTooltip

**Reply by KCN07** [reply]
it worked. Thanks a lot.
